lib/flutie/page_title_helper.rb

```diff
@@ -3,11 +3,13 @@ module PageTitleHelper
   def page_title(options = {})
     page_title = Flutie::PageTitle.new(options)
 
-    Flutie::PageTitlePresenter.new(
-      page_title.app_name,
-      content_for(page_title.page_title_symbol),
-      page_title.separator,
-      options[:reverse]
-    ).to_s
+    escape_once(
+      Flutie::PageTitlePresenter.new(
+        page_title.app_name,
+        content_for(page_title.page_title_symbol),
+        page_title.separator,
+        options[:reverse],
+      ).to_s,
+    )
```

Comment:
```suggestion
    ).html_safe
```

I think it's failing because of this change: https://github.com/rails/rails/pull/48265. It was released on Rails 7.1, so it makes sense that it's passing from that version onwards. Making it `html_safe` should avoid the double-escaping issue on older Rails.

Reply:
Good catch!